# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1259242795 -3600
#      Thu Nov 26 14:39:55 2009 +0100
# Branch release-3-2-x
# Node ID 050b8c0e1531191668a418300c3fba2c8347a2e4
# Parent  8166d596247ea3dffa8fa82e021be6e6c0d0ed01
correct previous patch

diff --git a/scripts/linear-algebra/dot.m b/scripts/linear-algebra/dot.m
--- a/scripts/linear-algebra/dot.m
+++ b/scripts/linear-algebra/dot.m
@@ -38,17 +38,17 @@ function z = dot (x, y, dim)
       x = x(:);
     endif
     if isvector (y)
       y = y(:);
     endif
     if (! size_equal (x, y))
       error ("dot: sizes of arguments must match");
     endif
-    z = sum(x .* conj (y));
+    z = sum(conj (x) .* y);
   else
     if (! size_equal (x, y))
       error ("dot: sizes of arguments must match");
     endif
-    z = sum(x .* conj (y), dim);
+    z = sum(conj (x) .* y, dim);
   endif
 
 endfunction
